packages/suite/src/utils/wallet/accountUtils.ts

```diff
@@ -704,11 +705,28 @@ export const getPendingAccount = (
     };
 };
 
-export const hasSignVerify = (account: Account) =>
-    !!NETWORKS.find(
-        ({ networkType, symbol, accountType, features }) =>
+export const hasNetworkFeatures = (
+    account: Account,
+    features: NetworkFeature | Array<NetworkFeature>,
+) => {
+    const networkConfig = NETWORKS.find(
+        ({ networkType, symbol, accountType }) =>
             networkType === account.networkType &&
             symbol === account.symbol &&
-            (accountType || 'normal') === account.accountType &&
-            (features || []).includes('sign-verify'),
+            (accountType || 'normal') === account.accountType,
     );
+
+    if (!networkConfig) {
+        return false;
+    }
+
+    let areFeaturesPresent: boolean;
+
+    if (Array.isArray(features)) {
+        areFeaturesPresent = features.every(feature => !!networkConfig.features?.includes(feature));
+    } else {
+        areFeaturesPresent = !!networkConfig.features?.includes(features);
+    }
+
+    return areFeaturesPresent;
```

Comment:
nothing wrong with this, just a small idea how to write it in a different way without using `let`:
```suggestion
    return ([] as NetworkFeature[])
        .concat(features)
        .every(feature => networkConfig.features?.includes(feature));
```

Reply:
5680bf19232d515b4216cde06def4f2d81f5297e